startup_runners/startup_runner.rb

```diff
@@ -0,0 +1,15 @@
+if /linux/ =~ RUBY_PLATFORM
```

Comment:
This needs the same hack as we just applied to the regular `iterations_runner.rb`. I.e. on JRuby we should call to `clock_gettime_monotonic()` in `libkruntime`.